### PR TITLE
rpcclient: expose the Node command for RPC API.

### PR DESCRIPTION
**Pull request by halseth**
The `AddNode` command is not particularly easy to use for managing connections, this PR exposes the more useful `Node` command for the `rpcclient`.

### CODE REVIEW

File: rpcclient/net.go [hunk 1]
```diff
@@ -63,6 +63,40 @@ func (c *Client) AddNode(host string, command AddNodeCommand) error {
...
+	return c.sendCmd(cmd)
+}
+
+// Node attempts to perform the passed node command on the host.
+// For example, it can be used to add or a remove a persistent peer, or to do
+// connect or diconnect a non-persistent one.
+//
+// The connectSubCmd should be set either "perm" or "temp", depending on
+// whether we are targetting a persistent or non-persistent peer. Passing nil
+// will cause the default value to be used.
```
Comment (cfromknecht) [inline rpcclient/net.go]: what is the default value?

File: rpcclient/net.go [hunk 2]
```diff
@@ -63,6 +63,40 @@ func (c *Client) AddNode(host string, command AddNodeCommand) error {
 	return c.AddNodeAsync(host, command).Receive()
 }
 
+// FutureNodeResult is a future promise to deliver the result of an NodeAsync
```
Comment (cfromknecht) [inline rpcclient/net.go]: s/an NodeAsync/a Node Async/

File: rpcclient/net.go (hunk 1 above)
Comment (halseth) [reply rpcclient/net.go]: Fixed.

File: rpcclient/net.go (hunk 2 above)
Comment (halseth) [reply rpcclient/net.go]: Fixed.

**Pull request merged**